components/eamxx/cime_config/testdefs/testmods_dirs/eamxx/prod/yaml_outs/eamxx_output.decadal.1dailyAVG_native.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML 1.1
 ---
-filename_prefix: scream_output.decadal.1dailyAVG_native.h
+filename_prefix: eamxx_output.decadal.1dailyAVG_native.h
```

Comment:
The general pattern should be [CASE_name].[model name].h[anystring].[date string].nc
so make this output.decadal.eamxx.h.1dailyAVG_native  then in your sed command replace "output.decadal" with "case_name".

Reply:
ok

Reply:
EAMxx should have .h. in the hist files. And we know that the next string is one between AVERAGE, INSTANT, MAX, MIN. So wouldn't this work?

```
<hist_file_extension>h\.(INSTANT|AVERAGE|MAX|MIN)\..*\.nc</hist_file_extension>
```

It would not clash with rhist files. I'm not sure what the issue is I guess?

Reply:
I thought that was part of one of our recent PRs.